meta-lmp-bsp/recipes-bsp/u-boot/u-boot-fio/imx8mmevk/lmp.cfg

```diff
@@ -47,7 +47,7 @@ CONFIG_BOOTCOUNT_LIMIT=y
 CONFIG_BOOTCOUNT_ENV=y
 CONFIG_BOOTDELAY=-2
 CONFIG_USE_BOOTCOMMAND=y
-CONFIG_BOOTCOMMAND="fatload mmc 2:1 ${loadaddr} /boot.itb; setenv verify 1; source ${loadaddr}; reset"
+CONFIG_BOOTCOMMAND="setenv verify 1; source 0x43800000; reset"
```

Comment:
@ricardosalveti @mike-scott could you please confirm that we're ok about using this address (that works on my side, but just to be sure that we are not going to use for something else)

Reply:
address LGTM